practice_app/google_shopping_api_key.txt

```diff
@@ -1 +1 @@
-Your rapidAPI:google_shopping key
```

Comment:
Could you move this file to the /practice_app/google_shopping/google_shopping_api_key.txt

Reply:
Sure, I have done it.